lua/indent_blankline/init.lua

```diff
@@ -275,6 +289,7 @@ local refresh = function()
         return virtual_text
     end
 
+    local prev_indent = nil
```

Comment:
just to stay consistent with the other variables
```suggestion
    local prev_indent
```